CIP-XXXX/README.md

```diff
@@ -0,0 +1,408 @@
+
+As wallets should already have this ability, we allow dApps to request that a transaction be sent through it. If the wallet accepts the transaction and tries to send it, it shall return the transaction id for the dApp to track. The wallet is free to return the `TxError` with code `Refused` if they do not wish to send it, or `Failure` if there was an error in sending it (e.g. preliminary checks failed on signatures).
+
+### Experimental API
+
+Multiple experimental namespaces are used:
+- under `api` (ex: `api.experimental.myFunctionality`).
+- under `cardanoMultisig.{walletName}` (ex: `window.cardanoMultisig.{walletName}.experimental.myFunctionality`)
+
+The benefits of this are:
+1. Wallets can add non-standardized features while still following the CIP30 structure
+1. dApp developers can use these functions explicitly knowing they are experimental (not stable or standardized)
+1. New features can be added to CIP30 as experimental features and only moved to non-experimental once multiple wallets implement it
+1. It provides a clear path to updating the CIP version number (when functions move from experimental -> stable)
+
+## Rationale: how does this CIP achieve its goals?
+
+See justification and explanations provided with each API endpoint.
+
+### Extensions
+
+Extensions provide an extensibility mechanism and a way to negotiate (possibly conflicting) functionality between a DApp and a wallet provider. There's rules enforced as for what extensions a wallet decide to support or enable. The current mechanism only gives a way for wallets to communicate their choice back to a DApp.
+
+We use object as extensions for now to leave room for adding fields in the future without breaking all existing interfaces. At this point in time however, objects are expected to be singleton.
+
+Extensions can be seen as a smart versioning scheme. Except that, instead of being a monotonically increasing sequence of numbers, they are multi-dimensional feature set that can be toggled on and off at will. This is a versioning "à-la-carte" which is useful in a context where:
+
+1. There are multiple concurrent standardization efforts on different fronts to accommodate a rapidly evolving ecosystem;
+2. Not everyone agrees and has desired to support every existing standard;
+3. There's a need from an API consumer standpoint to clearly identify what features are supported by providers.
+
+#### Namespacing Extensions
+
+By encouraging the explicit namespacing of each extension we aim to improve the usability of extensions for dApps. By allowing special cases where namespacing can be dropped we maintain good flexibility in extension design.
+
+## Path to Active
+
+### Acceptance Criteria
+
+- [ ] The interface is implemented and supported by various wallet providers.
+- [ ] The interface is used by dApps to interact with wallet providers. 
+  
+### Implementation Plan
+
+- [ ] Provide some reference implementation of wallet providers
+  - [leo42/BroClanWallet]
```

Comment:
(continuing previous comment) Hard to tell whether "BroClan Wallet" is a working implementation because of the ambiguous language... where do you still have yet to "provide" a reference implementation? in the CIP, or in software?

Reply:
yes I will be creating a reference implementation in BroClanWallet, its the next thing on the work agenda.

I am not sure how to mark that to make it more understandable. 

I expect the implementation to be completed in ~1 month.

Reply:
sure, I've updated the diff above with my suggestion... removing some extra language and unnecessary nesting.

If adoption by other wallet providers is a part of your Implementation Plan, please add some more checkboxes with the adoption plan as you see it. 🤓

Reply:
I am in contact with the Gamechanger and ADAO devs trying to get them onboard.